src/cli.js

```diff
@@ -169,6 +192,14 @@ function parseArgs(argv) {
       describe:
         "Local path or URL of schema to validate file against. If not supplied, we will attempt to find an appropriate schema on schemastore.org using the filename",
     })
+    .option("catalogs", {
+      type: "string",
+      alias: "c",
+      array: true,
+      default: [],
+      describe:
+        "Local paths of custom catalogs to use prior to schemastore.org",
+    })
```

Comment:
I can't think of a situation where it would make sense to pass both `--catalogs` and `--schema`. Can you? If not, maybe it would make them mutually exclusive with `.conflicts('schema', 'catalogs')`.?

Reply:
Fixed by dad46a136bc280a1863005a941aa9363e03e0a28

Yargs do not handle manage conflictual arguments with default value so I remove this behavior.